src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -339,8 +339,8 @@ def init_sync_d_queue_tables(db_conn):
     logger.debug("Queue name map:\n" + pprint.pformat(queue_name_map, indent=2))
 
     # Parse the queue_name_map and create the following maps:
-    # port_queues_map -> {"if_index : queue_index" : sai_oid}
-    # queue_stat_map -> {queue stat table name : {counter name : value}}
+    # port_queues_map -> {"if_name : queue_index" : sai_oid}
```

Comment:
>if_name [](start = 27, length = 7)

It should be "port_index"?

Reply:
Fixed this, it should be port_index and not port_name.